File: AD-HOC/2297/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        for (int test = 1, r = scanner.nextInt(); r > 0; test++, r = scanner.nextInt()) {
            System.out.printf("Teste %d\n", test);
            int totalA = 0, totalB = 0;
            while (r-- > 0) {
                totalA += scanner.nextInt();
                totalB += scanner.nextInt();
            }
            System.out.printf("%s\n\n", totalA > totalB ? "Aldo" : "Beto");
        }
        scanner.close();
    }
}

File: AD-HOC/2297/main.rb
test = 1
while (r = gets.to_i).positive?
  puts "Teste #{test}"
  total_a, total_b = 0, 0
  r.times do
    a, b = gets.split.map(&:to_i)
    total_a += a
    total_b += b
  end
  puts (total_a > total_b) ? "Aldo\n\n" : "Beto\n\n"
  test += 1
end
